DSCResources/MSFT_xNetworkTeam/MSFT_xNetworkTeam.psm1

```diff
@@ -0,0 +1,230 @@
+﻿if (Test-Path "${PSScriptRoot}\${PSUICulture}")
+{
+    Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xNetworkTeam.psd1 -BaseDirectory "${PSScriptRoot}\${PSUICulture}"
+}
```

Comment:
Someone correct me here if I'm wrong but if I have a language that's not found it's not going to load any strings correct?  Shouldn't we at least have an else statement to load english?